src/main/java/cubicchunks/CommonEventHandler.java

```diff
@@ -45,7 +45,7 @@ public void onWorldLoad(WorldEvent.Load evt) {
 		ICubicWorld world = (ICubicWorld) evt.getWorld();
 
 		WorldType type = evt.getWorld().getWorldType();
-		if(type instanceof ICubicWorldType){
+		if(type instanceof ICubicWorldType) {
```

Comment:
It would probably take less time to download and setup IDEA than to fix all of the formatting manually.